docs/sources/installation/helm/reference.md

```diff
@@ -1881,6 +1881,7 @@ null
     "accountKey": null,
     "accountName": null,
     "requestTimeout": null,
+    "useFederatedToken": false,
```

Comment:
See the note at line 9:  This file is autogenerated. Modify ../production/helm/loki/reference.md.gotmpl instead.

Reply:
My bad, thanks for pointing this out

Reply:
No worries, I just don't want doc updates to get overwritten and lost.  =)

Reply:
I am slightly confused. If I remove that change the doc will be updated with it since the helm-docs command will generate with it, as it comes out of the values.yaml. Is there something I am missing to update or should I just rebase without that file change?
using `helm-docs -t reference.md.gotmpl -o ../../../docs/sources/installation/helm/reference.md` just ends up with the same change above so I am not sure now if I should change something else

Reply:
It should be alright. We have a CI check for this.